extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -179,4 +185,21 @@ private function getLayout( $params ) {
 
 		return self::INFOBOX_LAYOUT_PREFIX . self::DEFAULT_LAYOUT_NAME;
 	}
+
+	private function moveFirstInfoboxToTop( $article ) {
+		$articleDecoded = json_decode( $article );
+
+		if ( !empty( $articleDecoded->content ) ) {
```

Comment:
Can you just do: ``$this->markers[0] . str_replace(array_keys( $this->markers )[0], '', $article->content)``